src/ai/backend/manager/services/resource_preset/service.py

```diff
@@ -259,12 +272,12 @@ async def check_presets(
                 remaining += ResourceSlot({k: Decimal(0) for k in known_slot_types.keys()})
                 sgroup_remaining += remaining
                 agent_slots.append(remaining)
-                per_sgroup[row["scaling_group"]]["remaining"] += remaining
+                per_sgroup[row["scaling_group"]][ResourceSlotState.AVAILABLE] += remaining
 
             # Take maximum allocatable resources per sgroup.
             for sgname, sgfields in per_sgroup.items():
                 for rtype, slots in sgfields.items():
-                    if rtype == "remaining":
+                    if rtype == ResourceSlotState.AVAILABLE:
```

Comment:
Changing the value that was being used as "remaining" directly to "available" seems like it might break version compatibility, is there no problem?